src/main/java/uk/gov/hmcts/divorce/citizen/notification/conditionalorder/EntitlementGrantedConditionalOrderNotification.java

```diff
@@ -116,8 +116,11 @@ public void sendToApplicant2Solicitor(final CaseData caseData, final Long id) {
     public void sendToApplicant2Offline(final CaseData caseData, final Long caseId) {
         if (!caseData.getConditionalOrder().hasOfflineCertificateOfEntitlementBeenSentToApplicant2()) {
             log.info("Sending certificate of entitlement letter to applicant 2 for case: {}", caseId);
-
-            certificateOfEntitlementPrinter.sendLetter(caseData, caseId, caseData.getApplicant2());
+            if (caseData.getApplicationType().isSole() && caseData.getApplicant2().isOffline()) {
```

Comment:
do we need the isOffline check if the NotificationDispatcher class checks this before calling this method?

Reply:
removed the condition